build.gradle

```
@@ -45,29 +68,131 @@ dependencies {
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.9'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.9.9'
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.9'
+    errorprone "com.uber.nullaway:nullaway:0.7.9"
+    errorprone "com.google.errorprone:error_prone_core:2.3.4"
+    spotbugsPlugins 'com.h3xstream.findsecbugs:findsecbugs-plugin:1.10.1'
 }
 
-repositories {
-    mavenCentral()
-    jcenter()
-    maven {
-        url 'https://dl.bintray.com/palantir/releases'
+// Configuration for the nullaway extension. The rest of the parameters must be
+// specified on the java-compile tasks. The only option that be specified here
+// is the name of the root package to be analysed.
+nullaway {
+    annotatedPackages.add("com.nostod")
```

```suggestion
    annotatedPackages.add("com.nosto")
```

Looks like a typo here.

---

Fixed.